### ISSUE TITLE
Talk page integration?

### ISSUE THREAD

**Issue by chrisgatewood** [jmilldotdev/obsidian-wikipedia#3]
Hi. I was wondering if it'd be possible to have an option for any relevant metadata from the Wiki article's talk page? 

Love the plugin. Thanks for the great work! It's streamlined my workflow quite a bit.

**Comment by jmilldotdev** [comment]
Hey, glad you're enjoying it.

Unfortunately Wikipedia's public API is a bit janky and hard to parse, due to the fact that it's all returned as pseudo-markdown.

If you can figure out a way to do this I'm happy to take pull requests.
